src/auth.rs

```diff
@@ -2,46 +2,59 @@
 // SPDX-License-Identifier: Apache-2.0
 //! Client app authentication data
 use parsec_interface::requests::{request::RequestAuth, AuthType};
-use parsec_interface::secrecy::{ExposeSecret, Secret};
 
 /// Authentication data used in Parsec requests
 #[derive(Clone, Debug)]
-pub enum AuthenticationData {
+pub enum Authentication {
     /// Used in cases where no authentication is desired or required
     None,
     /// Data used for direct, identity-based authentication
     ///
-    /// The app name is wrapped in a [`Secret`](https://docs.rs/secrecy/*/secrecy/struct.Secret.html).
-    /// The `Secret` struct can be imported from
-    /// `parsec_client::core::secrecy::Secret`.
-    AppIdentity(Secret<String>),
+    /// Warning: Systems using direct authentication require extra measures
+    /// to be as secure as deployments with other authentication mechanisms.
+    /// Please check the
+    /// [Parsec Threat Model](https://parallaxsecond.github.io/parsec-book/parsec_security/parsec_threat_model/threat_model.html)
+    /// for more information.
+    AppIdentity(String),
```

Comment:
Do we keep `AppIdentity` here then and not use `Direct` 😬 ?

Reply:
Oh, indeed, forgot about that